test/features/build_webhooks.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Configure Webhooks Triggers in Kogito Build
```

Comment:
Wouldn't it have more sense to put the scenario into https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/features/deploy_kogito_build.feature ?

Reply:
Moved